Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access_README.md

```diff
@@ -0,0 +1,62 @@
+Investigates a Cortex XDR incident containing First SSO access from ASN in an organization
+ or First successful SSO connection from a country in an organization.
+
+The playbook executes the following:
+- IP and User Enrichment.
+- User Investigation - Using 'User Investigation - Generic' sub-playbook.
+- First SSO Access investigation - Using 'Cortex XDR - First SSO Investigation' sub-playbook.
```

Comment:
should remove this line

Reply:
also in the playbook description (playbook yml)

Reply:
Done.